Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -187,6 +187,13 @@ TextPosition TextEditor::text_position_at_content_position(Gfx::IntPoint content
             column_index = start_of_line;
             int glyph_x = 0;
 
+            // We're outside the text on the top, put cursor at column 0 and line 0.
+            if (position.y() <= 0) {
+                column_index = 0;
+                line_index = 0;
+                return IterationDecision::Break;
+            }
```

Comment:
i see it works but it still makes me wonder; iʼd guess iʼd go for `y ≤ 0` yielding `line → 0` and if `y < 0` then additionally `column → 0`… 🤔

Reply:
I don't quite understand why this happens, but once y is -1, -2... column gets set to line length.

Reply:
maybe we should not rely on that to avoid mistakes when this mysterious logic changes? 🤔

but thatʼs on you (and maintainers). 😄